WooCommerce/WooCommerceTests/Mocks/MockProductReviewFromNoteParcel.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import XCTest
+@testable import Yosemite
+
+
+final class MockProductReviewFromNoteParcel {
```

Comment:
We probably need to rename some of these `Mock*` classes at some point. They're misleading. It took me a while to understand why we call them mocks. In reality, some are just builders of objects. 🤷

Reply:
It looks like the future direction for these `Mock*` is to replace them with `Fakeable` https://github.com/woocommerce/woocommerce-ios/issues/3823 - however, only Networking models are supported right now and `ProductReviewFromNoteParcel` is in Yosemite. I asked about including Yosemite models in the issue, maybe we can add support for Yosemite in the future 😃 

Though for this case, we probably want to keep this class that creates `metaAsData` for the meta site ID. Maybe we can rename this class to like `ProductReviewFromNoteParcelFactory` to differentiate it from other `Mock*` classes?

Reply:
Btw, I tried generating Copiable for `Note` to replace the `Note` generation in this class. But because `Note` has a few properties that are not in the designated initializer like `metaAsData` and `timestampAsDate`, the generated `copy` function isn't calling the designated initializer. We can probably update the designated initializer to include all properties and mark the [current initializer](https://github.com/woocommerce/woocommerce-ios/blob/fb25492561c0d5bc16ab148b314969583880b886/Networking/Networking/Model/Note.swift#L100-L113) as a convenience initializer?

Reply:
> Yeah, it's certainly worth a try. But up to you if you feel that's valuable to do right now. Note is probably not used a lot to warrant the effort. 🤷

Going to do it in a separate issue/PR! https://github.com/woocommerce/woocommerce-ios/issues/5060

Reply:
> *Factory sounds fine by me!

Renamed in https://github.com/woocommerce/woocommerce-ios/pull/5044/commits/5626cd846c2647073348cb56cb65b5b05e13a93f